src/Primitives/rdMatrix.c

```diff
@@ -854,7 +855,7 @@ void rdMatrix_PostMultiply34(rdMatrix34* mat1, rdMatrix34* mat2)
 void rdMatrix_PreRotate34(rdMatrix34* out, rdVector3* rot)
 {
     rdMatrix34 tmp;
-    rdMatrix_BuildRotate34(out, &tmp);
```

Comment:
Why comment this ? I believe there is a call to BuildRotate

Reply:
tmp is of type rdMatrix34, but rdMatrix_BuildRotate34's second parameter expects a rdVector3

As I was just trying to get it compiling and this code wasn't loaded, I didn't try to see how I could convert an rdMatrix34 to an rdVector3 (well other than doing a matrix multiply to select a row vector)

Reply:
Indeed a mistake on my end. Thanks !